specreduce/background.py

```diff
@@ -32,9 +32,9 @@ class Background(_ImageParser):
     ----------
     image : `~astropy.nddata.NDData`-like or array-like
         image with 2-D spectral image data
-    traces : trace, int, float (single or list)
+    traces : List, `tracing.Trace`, int, float
```

Comment:
i think this is one of the RTD complaints. should be `~specreduce.tracing.Trace`, i believe.